docker-compose.yml

```diff
@@ -1,31 +1,40 @@
-version: '2.0'
+version: '3.0'
 services:
-  es1:
-    # image: divante/vue-storefront-elasticsearch:5.6.9
+  elasticsearch:
+    container_name: elasticsearch
+    # image: divante/vue-storefront-elasticsearch:5.6.10
     build: docker/elasticsearch/
-    container_name: es1
-    env_file: docker/elasticsearch/default.env
     volumes:
+      - ./docker/elasticsearch/config/elasticsearch.yml:/usr/share/elasticsearch/config/elasticsearch.yml:ro
       - 'esdat1:/usr/share/elasticsearch/data'
-    networks:
-      default:
-        aliases:
-          - elasticsearch
     ports:
       - '9200:9200'
-    ulimits:
-      memlock:
-        soft: -1
-        hard: -1
+      - '9300:9300'
+    environment:
+      ES_JAVA_OPTS: "-Xmx512m -Xms512m"
+    networks:
+      - elk
+
   kibana:
-    # image: divante/vue-storefront-kibana:5.6.9
+    # image: divante/vue-storefront-kibana:5.6.10
     build: docker/kibana/
-    env_file: docker/kibana/default.env
+    volumes:
+      - ./docker/kibana/config/:/usr/share/kibana/config:ro
     ports:
       - '5601:5601'
+    networks:
+      - elk
+    depends_on:
+      - elasticsearch
+
   redis:
     image: 'redis:4-alpine'
     ports:
       - '6379:6379'
+
+networks:
```

Comment:
What is the justification for instantiating an isolated network for ES and Kibana?

In this case, will the container with the VS runtime environment (docker-compose.nodejs.yml) still have connectivity to the service via the hostname "elasticsearch"?

Reply:
I prefer putting Elasticsearch and Kibana together because they shouldn't be separated from each other, Kibana is just a tool for Elasticsearch

Reply:
Ok, but once again - will the container with the VS runtime environment (_docker-compose.nodejs.yml_) still have connectivity to the service via the hostname "elasticsearch"?

If you isolate ES in its own network, it won't be a member of the _default_ network (as "vue-storefront-api_app" container).

Reply:
Yeah, that's better not to create any network at all, and removing:
```
 networks:	
      default:	
        aliases:	
          - elasticsearch
```
Since it's not used either